style.css

```diff
@@ -0,0 +1,60 @@
+*{
+  margin:0;
+  padding: 0;
+
+}
+div.nav{
+  display: flex;
+  justify-content: space-between;
+  padding: 20px;%;
+  width: 100%;
+}
+div.main{
+  display: flex;
+  padding: 20px;
+  width:100%;
+}
+div.searchPart{
+  width:40%;
+}
+form input{
+  width:70%;
+}
+div.signIn{
+  width:10%;
+
+
+}
+div.video{
+  box-sizing: border-box;
```

Comment:
Please better create a class for this rule, or set it to a father element instead of repeating it so much. For example 
`.borderBox { box-sizing: border-box; }`